### DISCUSSION TITLE
为什么vue2版本 2.12.54版本的vant的cascader不增加 disabled 禁用选项的适配呢

### DISCUSSION THREAD

**Discussion by shenxiaobu** [youzan/vant#11818] [Q&A]
2.12.54版本cascader数据配置了disabled不见效。查看源码，v3和v4有此配置，而唯独v2没有 为啥不对v2更新下呢

**Comment by chenjiahan** [comment]
v2 的文档里有说明：

![image](https://user-images.githubusercontent.com/7237365/236133039-44046135-4067-4522-90cd-f23e0609edc2.png)
